pallets/amendments/src/lib.rs

```diff
@@ -20,8 +20,9 @@
 
 //! An amendment module instance manages amendments to the chain. There could be a security
 //! delay configured along with a veto capability.
-
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
do you need that flag? It is normally showing up on top of the `benchmarking.rs` file

Reply:
Indeed not required, removed, Done :+1: